### DISCUSSION TITLE
String.prototype.replaceAll Polyfill not working

### DISCUSSION THREAD

**Discussion by daksh-sagar** [swc-project/swc#5366] [Q&A] [answered]
my webpack config looks like this 

```
{
      test: [/\.js$/, /\.es6$/, /\.jsx$/],
      exclude: /node_modules/,
      use: {
        loader: 'swc-loader',
        options: {
          jsc: {
            parser: {
              syntax: 'ecmascript',
              jsx: true,
            },
          },
          env: {
            targets: {
              node: '14.17',
            },
            mode: 'usage',
            coreJs: 3,
          },
        },
      },
    },
```

on running the built code, i get `items.replaceAll is not a function`. Shouldn't this have been properly polyfilled? Is there something wrong with my configuration?

p.s. - this used to work with babel. I'm trying to migrate the project from babel to swc

@swc-core version - 1.2.222
swc-loader version - 0.2.3
Node version - 14.17.5
Webpack version - 5.39.1

**Comment by zloirock** [comment, marked as answer]
`coreJs: '3.24'`

**Reply by daksh-sagar** [reply]
This worked,  thanks a lot !

also one more thing - what's the difference b/w specifying a `target` inside `jsc` config and specifying targets inside `env` object? Both of them are related to managing polyfills right?
